### DISCUSSION TITLE
Page Prefetching Plugin

### DISCUSSION THREAD

**Discussion by FLMMFL** [GetPublii/Publii#1603] [General]
Hi all, just seen this page prefetching module and looks cool. I use cloudflare and cache content there, would I still benefit from this module? and if so , have you any specific recommendations in the config bearing in mind I am using Cloudflare?    thanks in advance for your help.

**Comment by FLMMFL** [comment]
has anyone installed and tested this , that also has Cloudflare yet?

**Reply by AKickIndieNutz** [reply]
Hi yes my site uses Cloudflare. I installed the prefetch plug in and then synced my site up. I noticed after running Google search console that the performance stats seemed not to be as good as they were without the plug in. I didn't play around much more with it tbh and I uninstalled the plug in and re-synced without it. The performance seems better without. Guess this doesn't help answer your question though, maybe someone else on here can give us some pointers in configuring it?

**Reply by FLMMFL** [reply]
thank you, that is very helpful, because the last thing I want to do is install and create problems, better if I stay clear of it for now. Appreciate the input, have a good day.
